### PR TITLE
Private DNS Zones- Bug Fix1485

### PR DESCRIPTION
**Pull request by rozkurt**
## Overview/Summary

This PR is targeting bug report https://github.com/Azure/Enterprise-Scale/issues/1485 and it's adding new built-in policies into "Deploy-Private-DNS-Zones" initiative and assignment. All the new policy names are documented on `Whats-New` file.

## This PR fixes/adds/changes/removes

1. Updated `Deploy-Private-DNS-Zones.json`
2. Updated `DINE-PrivateDNSZonesPolicyAssignment.json`
3. Updated `eslz-portal.json`

### Breaking Changes

1. N\A

## Testing Evidence

Ekran görüntüsü 2024-02-21 114252
Ekran görüntüsü 2024-02-21 115305
Ekran görüntüsü 2024-02-21 115446

#### Azure Public

[Deploy To Azure

## As part of this Pull Request I have

- [x] Checked for duplicate [Pull Requests](https://github.com/Azure/Enterprise-Scale/pulls)
- [x] Associated it with relevant [issues](https://github.com/Azure/Enterprise-Scale/issues), for tracking and closure.
- [x] Ensured my code/branch is up-to-date with the latest changes in the `main` [branch](https://github.com/Azure/Enterprise-Scale/tree/main)
- [x] Performed testing and provided evidence.
- [x] Ensured [contribution guidance](https://github.com/Azure/Enterprise-Scale/wiki/ALZ-Contribution-Guide) is followed.
- [x] Updated relevant and associated documentation.
- [x] Updated the ["What's New?"](https://github.com/Azure/Enterprise-Scale/wiki/Whats-new) wiki page (located: `/docs/wiki/whats-new.md`)

### CODE REVIEW

File: docs/wiki/Whats-new.md [hunk 1]
```diff
@@ -4,6 +4,8 @@
   - [March 2024](#march-2024)
   - [February 2024](#february-2024)
   - [AMA Update for the Portal Accelerator](#ama-update-for-the-portal-accelerator)
+  - [March 2024](#march-2024)
+  - [AMA Update for the Portal Accelerator](#AMA-Update-for-the-Portal-Accelerator)
```
Comment (jtracey93) [inline docs/wiki/Whats-new.md:8]: duplicated from lines 4 & 5?

**Review by jtracey93 — CHANGES_REQUESTED** [review]
just a couple of changes please @rozkurt 

cc: @Springstone

File: docs/wiki/Whats-new.md (hunk 1 above)
Comment (rozkurt) [reply docs/wiki/Whats-new.md]: @jtracey93 I just removed duplicated lines!

**Review by Springstone — APPROVED** [review]
LGTM

**Review by jtracey93 — APPROVED** [review]

**Pull request merged**